src/app/auth/page.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const AuthPage: React.FC = () => {
+  return (
+    <main className="flex min-h-screen flex-col items-center justify-between p-24">
```

Comment:
@ticogarcia I copied this style from the starter page, which makes the component occupy the entire page height with `min-h-screen`. Fixed. Merge when able.